roles/container_runtime/tasks/systemcontainer_crio.yml

```diff
@@ -53,6 +53,8 @@
     name: "cri-o"
     image: "{{ l_crio_image }}"
     state: latest
+    values:
+    - "ADDTL_MOUNTS={{',' + (container_runtime_crio_additional_mounts | join(',')) if container_runtime_crio_additional_mounts != [] else '' }}"
```

Comment:
My preference is to have users provide a list of dictionaries instead of a list of strings.  See lib_utils_oo_l_of_d_to_csv and roles/openshift_node/tasks/node_system_container.yml line 33.

Reply:
I can try that out.  I think I also prefer native yaml objects and converting them.